src/main/java/com/exasol/releaserobot/main/Runner.java

```diff
@@ -17,7 +28,7 @@
      *
      * @param args arguments
      */
-    public static void main(final String[] args) {
+    public static void main(final String[] args) throws GitHubException {
```

Comment:
Throwing an exception here is not a good idea, cause in the main method it's not helpful to the user.

Instead for such cases you want to use a runtime exception.

As a general guideline you want use checked exceptions for recoverable errors and to provide the user of your API more information about what could happen so he/she might recover form it.
Otherwise, use runtime exception and let them bubble up.

This can also be done a following refactor too.